lib/zhuyin/parsers.ex

```diff
@@ -0,0 +1,99 @@
+defmodule Zhuyin.Parsers do
+  @moduledoc false
+  # Parsing logic for Zhuyin strings
+
+  import NimbleParsec
+  alias Pinyin.Parsers.Utils, as: Utils
+  alias Pinyin.Parsers.Wordlist, as: Wordlist
+
+  # Empty tone must be last for parsing to work
+  zhuyin_tones = ["˙", "ˊ", "ˇ", "ˋ", ""]
```

Comment:
I feel it is a bit strange to add the empty string as the final tone here. Would it not make more sense to just assume no tone (as the tone parser is optional anyway), like in the pinyin parser?